### PR TITLE
Add the Web Payments collection to /learn

### PR DESCRIPTION
**Pull request by mihajlija**
A patch for #2907

cc @agektmr @kaycebasques

### CODE REVIEW

File: src/images/collections/payment.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...d="m136.5 78.5 11.5 11.5" stroke-dasharray="2 2" stroke-linecap="round"/><path d="m130.5 80.5 11.5 11.5" stroke-dasharray="2 2" stroke-linecap="round"/><path d="m125.5 82.5 11.5 11.5" stroke-dasharray="2 2" stroke-linecap="round"/><path d="m120.5 82.5 11.5 11.5" stroke-dasharray="2 2" stroke-linecap="round"/><path d="m115.5 81.5 11.5 11.5" stroke-dasharray="2 2" stroke-linecap="round"/><path d="m145.5 69.5 11.5 11.5" stroke-dasharray="2 2" stroke-linecap="round"/><path d="m147.5 64.5 11.5 11.5" stroke-dasharray="2 2" stroke-linecap="round"/><path d="m148.5 60.5 11.5 11.5" stroke-dasharray="2 2" stroke-linecap="round"/><path d="m149.5 56.5 11.5 11.5" stroke-dasharray="2 2" stroke-linecap="round"/><path d="m149.5 53.5 11.5 11.5" stroke-dasharray="2 2" stroke-linecap="round"/><path d="m148.5 49.5 11.5 11.5" stroke-dasharray="2 2" stroke-linecap="round"/><path d="m147.5 45.5 11.5 11.5" stroke-dasharray="2 2" stroke-linecap="round"/><path d="m141.5 74.5 11.5 11.5" stroke-dasharray="2 2" stroke-linecap="round"/><path d="m94.5 22v12m-5.5-6h11" stroke-linecap="round"/><path d="m141.5 30v12m-5.5-6h11" stroke-linecap="round"/><path d="m90.5 69v12m-5.5-6h11" stroke-linecap="round"/></g></svg>
```
Comment (robdodson) [inline src/images/collections/payment.svg:1]: Just checking — have you already run this through svgomg to optimize it?

File: src/site/content/en/payments/payments.11tydata.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    slug: 'payments',
+    cover: '/images/collections/payments.svg',
+    title: 'Web Payments',
+    updated: 'May 25, 2020',
+    description: `Easy way to implement payments on the web.`,
+    overview: `In this section you'll learn how to use Web Payments APIs.`,
+    topics: [
+      {
+        title: 'Web Payments',
+        pathItems: ['setting-up-a-payment-method'],
```
Comment (robdodson) [inline src/site/content/en/payments/payments.11tydata.js:16]: Is the plan to add more docs to this learning path before we land it? I'm not sure if we want to ship a learning path with only one item in it.

File: src/images/collections/payment.svg (hunk 1 above)
Comment (mihajlija) [reply src/images/collections/payment.svg:1]: Haven't, will do.

File: src/site/content/en/payments/payments.11tydata.js (hunk 2 above)
Comment (mihajlija) [reply src/site/content/en/payments/payments.11tydata.js:16]: There'll be 3 or 4 items here by Monday, another 3-4 coming in a couple of weeks later.

Comment (robdodson) [reply src/site/content/en/payments/payments.11tydata.js:16]: ok sounds good :)

**Pull request merged**